packages/regl-worldview/src/Worldview.js

```diff
@@ -249,15 +251,14 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
   }
 
   render() {
-    const { width, height, showDebug, keyMap } = this.props;
+    const { width, height, showDebug, keyMap, style } = this.props;
     const { worldviewContext } = this.state;
-    const style = { width, height };
 
     return (
-      <React.Fragment>
+      <div style={{ position: "relative", overflow: "hidden", ...style }}>
         <CameraListener cameraStore={worldviewContext.cameraStore} keyMap={keyMap}>
           <canvas
-            style={style}
+            style={{ width, height, maxWidth: "100%", maxHeight: "100%" }}
```

Comment:
does the `maxWidth: "100%", maxHeight: "100%"` do anything?

Reply:
While playing locally, I noticed the canvas width and height can go over the container depending on the parent's layout. Adding this will ensure that the canvas doesn't overflow.